sdk/src/wallet/operations/transaction/prepare_transaction.rs

```diff
@@ -62,6 +66,8 @@ where
                 RemainderValueStrategy::CustomAddress(address) => Some(address.clone()),
             });
 
+        let issuer_id = issuer_id.into();
```

Comment:
Related to https://github.com/iotaledger/iota-sdk/pull/1961/files#r1482572913, probably better to set the issuer_id here already and add a field in the TransactionOptions where one can specify it as well?
So if provided issuer_id, use it
If not provided, use from TransactionOptions and if there also not provided, use first account

Reply:
So now you moved the isser_id in the TransactionOptions, but if the user provides nothing, then we also allot nothing?

Reply:
yes. I can still make it default to the first account address, just not 100% sure where

Reply:
Probably in prepare_transaction(), just wondering if we want to allow having None in the wallet

Reply:
I did it in select inputs instead because that's where we actually use it. And it's fine to have None in the wallet, because then you just won't allot mana and it will have to come from previously allotted mana instead.

Reply:
But it will end up calling select_inputs(), so there is no way it ends up without issuer_id?

Reply:
If the account doesn't have one, then it won't

Reply:
Ah yeah, guess that's fine for now